superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -502,21 +506,76 @@ export default class ResultSet extends React.PureComponent<
   }
 
   renderRowsReturned() {
-    const { results, rows, queryLimit } = this.props.query;
+    const { results, rows, queryLimit, limitingFactor } = this.props.query;
     const limitReached = results?.displayLimitReached;
+    let limitMessage;
+    const appContainer = document.getElementById('app');
+    const bootstrapData = JSON.parse(
+      appContainer?.getAttribute('data-bootstrap') || '{}',
```

Comment:
this is in redux already.. can you get it from there?

Reply:
Of course! I did it this way because this is how user was accessed earlier in the file:
https://github.com/apache/superset/blob/master/superset-frontend/src/SqlLab/components/ResultSet.tsx#L321

should I amend both?

Reply:
oh, I see, yes that would be great if you could update both.. thanks!